### ISSUE TITLE
Using special comment syntax with /*!90000 */ caused TiCDC to report an error.

### ISSUE THREAD

**Issue by SE-Bin** [pingcap/tiflow#3755]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
I found that creating a table by using special comment syntax with `/*!90000 */` caused TiCDC to report an error.
The following step:
1.  In TiDB : 
```sql
create table cdc_test (id varchar(10) primary key ,c1 varchar(10)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin/*!90000  SHARD_ROW_ID_BITS=4 PRE_SPLIT_REGIONS=3 */
```

### 2. What did you expect to see? (Required)
TiCDC can sync DDL to downstream successfully.
### 3. What did you see instead (Required)
TiCDC cause the error 
```
[WARN] [mysql.go:208] ["execute DDL with error, retry later"] [query="create table cdc_test (id varc
har(10) primary key ,c1 varchar(10)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin/*!90000 /*T! SHARD_ROW_ID_BITS=4 PRE_SPLIT_REGIONS=3 */ */"] [error="[CDC:ErrMySQLTxnError]Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 182 near \"*/\" "]
```
### 4. What is your TiDB version? (Required)
TiDB & TiCDC: v5.0.3

**Comment by hi-rustin** [comment]
Is your downstream TiDB or MySQL?

**Comment by SE-Bin** [comment]
In my case ，downstream is TiDB.

**Comment by maxshuang** [comment]
@sdojjy  The pr related to this issue can rewrite the follow DDL to compatible with MySQL?

[2021/12/17 11:24:16.467 +08:00] [WARN] [mysql.go:218] ["execute DDL with error, retry later"] [query="/* init */ create table t (id int primary key, v int) shard_row_id_bits 4;"] [error="[CDC:ErrMySQLTxnError]Error 8200: Unsupported shard_row_id_bits for table with primary key as row id"] [errorVerbose="[CDC:ErrMySQLTxnError]Error 8200: Unsupported shard_row_id_bits for table with primary key as row

**Comment by sdojjy** [comment]
@maxshuang  #3845  will rewrite 
`/* init */ create table t (id int primary key, v int) shard_row_id_bits 4;`  to `CREATE TABLE `t` (`id` INT PRIMARY KEY,`v` INT) /*T! SHARD_ROW_ID_BITS = 4 */`

**Comment by sdojjy** [comment]
TiCDC introduced this bug in v4.0.0, and it is fixed since v5.2.0.
